controller/invitation.go

```diff
@@ -73,3 +81,29 @@ func (c *InvitationController) CreateInvite(ctx *app.CreateInviteInvitationConte
 
 	return ctx.Created()
 }
+
+func (c *InvitationController) AcceptInvite(ctx *app.AcceptInviteInvitationContext) error {
+	currentIdentity, err := login.LoadContextIdentityIfNotDeprovisioned(ctx, c.app)
+	if err != nil {
+		return jsonapi.JSONErrorResponse(ctx, err)
+	}
+
+	acceptCode, err := uuid.FromString(ctx.AcceptCode)
+	if err != nil {
+		log.Error(ctx, map[string]interface{}{
+			"err": err,
+		}, "failed to accept invitation, invalid code")
+
+		return jsonapi.JSONErrorResponse(ctx, errors.NewBadParameterErrorFromString("AcceptCode", ctx.AcceptCode, "accept code is not a valid uuid"))
+	}
+
+	c.app.InvitationService().Accept(ctx, currentIdentity.ID, acceptCode)
+
+	log.Debug(ctx, map[string]interface{}{
+		"accepting-user-id": *currentIdentity,
+		"accept-code":       ctx.AcceptCode,
+	}, "invitation accepted")
+
+	ctx.ResponseData.Header().Set("Location", c.config.GetInvitationAcceptedRedirectURL())
```

Comment:
I missed passing on this review feedback to you @sbryzak - sorry about that.

When there's a success in the 'Accept invite' controller, we redirect to `c.config.GetInvitationAcceptedRedirectURL()`
When there's an error in the 'Accept invite' controller, we still *redirect* to `c.config.GetInvitationAcceptedRedirectURL()+"?error="+errorString`  but with this extra error parameter in the end.

We need to do a 
```
_,err = c.app.InvitationService().Accept(ctx, currentIdentity.ID, acceptCode)
```
and then if `err != nil` we need to pass the `error` URL parameter. That way, the UI can show the user a failure/success.

Could you please make the above changes, and also update the tests to validate the presence of the "Location" header, with an `error` URL parameter conditionally?

Reply:
Done